--- a/generate-db.js
+++ b/generate-db.js
@@ -208,8 +208,10 @@
 
         const packageName = fileMap.get(join(TERMUX_PREFIX.substring(1), path));
         if (packageName === undefined) {
-          console.error(`Package name not found for path: ${path}`);
-          process.exit(1);
+          console.warn(
+            `WARNING: Package name not found for path: ${path}. Skipping...`,
+          );
+          return;
         }
         if (!binMap.has(packageName)) {
           binMap.set(packageName, []);
